documentation/docs/07-misc/02-testing.md

```diff
@@ -246,6 +248,82 @@ test('Component', async () => {
 
 When writing component tests that involve two-way bindings, context or snippet props, it's best to create a wrapper component for your specific test and interact with that. `@testing-library/svelte` contains some [examples](https://testing-library.com/docs/svelte-testing-library/example).
 
+_With Storybook_
+
+[Storybook](https://storybook.js.org?ref=svelte-docs) is the industry standard for developing and documenting UI components, and it can also be used to test your components. A storybook is made up of [stories](https://storybook.js.org/docs/writing-stories?ref=svelte-docs&renderer=svelte)—isolated examples of your components in different states and with simulated interactions. These stories can also function as [component tests](https://storybook.js.org/docs/writing-tests?ref=svelte-docs&renderer=svelte), with no additional code needed. They're run with Vitest's browser mode, which renders your components in a real browser for the most realistic testing environment.
```

Comment:
I don't think storybook needs an intro here, this is the testing section and the focus should be on that. Use lots of links into canonical storybook documentation so users can learn more about storybook specific concepts, a brief example of svelte native story format and thats it. Shouldn't take more than a few lines total.

The current block about jsdom+vitest needs an overhaul as well, linking to sveltest.dev instead (sorry if you took that as a template for your work)